.github/workflows/update-docs-version.yml

```diff
@@ -0,0 +1,30 @@
+name: Update docs version
+
+on:
+    release:
+        types: [published]
+
+jobs:
+    build:
+        runs-on: ubuntu-18.04
+        steps:
+            -   uses: actions/checkout@v1
+            -   uses: webfactory/ssh-agent@ea39f521c5e59e2e0a73686497d303bb2b499e03
```

Comment:
WDYT about using https://github.com/ad-m/github-push-action or similar action?

Reply:
This action, and others like it, all seem to use access tokens. AFAICT there is no way for us to create an access token that belongs to an organisation or repo; it's always a personal access token associated with an individual user.

I'd prefer not to use a personal access token for an automation, and deploy keys (SSH based) seem like the only way to create a credential that is effectively owned by the repository (I have discarded the private key after setting it as a secret for the repo).

Reply:
the default access token that GHA assigns to every build automatically as a secret allows us to push to the same repo AFAIK

Reply:
I thought that disallowed pushes... will check.

Reply:
Actually, looks like I might have been wrong. Will give it a try.